lib/LTP/utils.pm

```diff
@@ -181,38 +188,58 @@ sub check_kernel_taint {
         'ACPI table overridden by user',
         'Kernel issued warning',
         'Staging driver was loaded',
-        undef,    # platform firmware bug workaround, ignore
-        undef,    # out-of-tree module, ignore
+        'Workaround for platform firmware bug',
+        'Out of tree module was loaded',
         'Unsigned module was loaded',
         'Soft lockup occurred',
-        undef,    # livepatch, ignore
-        'Auxiliary taint',
-        undef,    # kernel built with struct randomization, ignore
-        'In-kernel test has been run'
+        'Kernel was live patched',
+        'Externally supported module was loaded or auxiliary taint',
+        'Kernel was built with struct randomization',
+        'In-kernel test has been run',
+        # currently unused taint flags
+        undef, undef, undef, undef, undef, undef, undef, undef, undef, undef,
+        undef, undef,
+        'Unsupported module was loaded'
```

Comment:
nit: For `'Unsupported module was loaded'`  I would write as a comment the number of the last (`1 << 31`), because the number of undef is quite big. Or using `31` as a key(others are sequential) and shift by key  would be more readable.

Reply:
Fixed, unsupported module message is now added separately to index 31.